### PR TITLE
Create bind for RecyclerView-FastScroll

### PR DESCRIPTION
**Pull request by 4brunu**

### CODE REVIEW

**Review by moljac — CHANGES_REQUESTED** [review]
please could you rename folder from `RecyclerView-FastScroll` to `RecyclerView.FastScroll` to keep it somehow consistent with .NET naming.

**Comment by 4brunu** [comment]
Done, I also updated the monoandroid to version 11.0, but now the build is failing

**Comment by moljac** [comment]
> Done, I also updated the monoandroid to version 11.0, but now the build is failing

please revert to 9.0

and build is failing because of changes for .NET 6. We are working on that issue

**Comment by 4brunu** [comment]
Done 👍

**Comment by moljac** [comment]
@4brunu

Error:

```
Script Arguments:
  Root directory: /Users/runner/work/1/s
  Root output directory: output
  Valid package namespaces: Xamarin, Mono, SkiaSharp, HarfBuzzSharp, mdoc, Masonry, GoogleGson, AndroidEasingFunctions, Square, JakeWharton.Picasso2OkHttp3Downloader, RecyclerViewAnimators, GoogleGson

Found 1 NuGet packages to validate.
Verifying NuGet metadata of /Users/runner/work/1/s/output/RecyclerView.FastScroll.2.0.1.nupkg...
NuGet metadata validation failed for /Users/runner/work/1/s/output/RecyclerView.FastScroll.2.0.1.nupkg:
The package does not start with any of the following namespaces: [ 'Xamarin', 'Mono', 'SkiaSharp', 'HarfBuzzSharp', 'mdoc', 'Masonry', 'GoogleGson', 'AndroidEasingFunctions', 'Square', 'JakeWharton.Picasso2OkHttp3Downloader', 'RecyclerViewAnimators', 'GoogleGson' ]
```

please add Xamarin as nuget package prefix.

I tend to make everything with prefix (nuget packageid, namespaces and assemblyname)

**Comment by 4brunu** [comment]
@moljac Can you please confirm if https://github.com/xamarin/XamarinComponents/pull/1186/commits/eedefc63df40e00232a212d9556e47732ad486b7 does what you asked for?

File: Android/RecyclerView.FastScroll/source/RecyclerView.FastScroll/RecyclerView.FastScroll.csproj [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <AssemblyName>RecyclerView.FastScroll</AssemblyName>
+    <RootNamespace>RecyclerView.FastScroll</RootNamespace>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>RecyclerView.FastScroll</PackageId>
```
Comment (moljac) [inline Android/RecyclerView.FastScroll/source/RecyclerView.FastScroll/RecyclerView.FastScroll.csproj]: This is nuget ID (needs Xamarin prefix)

**Review by moljac — CHANGES_REQUESTED** [review]
Xamarin prefix

File: Android/RecyclerView.FastScroll/source/RecyclerView.FastScroll/RecyclerView.FastScroll.csproj (hunk 1 above)
Comment (4brunu) [reply Android/RecyclerView.FastScroll/source/RecyclerView.FastScroll/RecyclerView.FastScroll.csproj]: Done, thanks 👍

**Review by moljac — APPROVED** [review]
LGTM

**Pull request merged**